### ISSUE TITLE
Display subtitles from SRT file

### ISSUE THREAD

**Issue by jo-elimu** [elimu-ai/filamu#5]
## Description

If the video comes with attached with subtitles, extract the words from the SRT file and display at the bottom of the video.

Sample format:

```srt
1
00:00:00,080 --> 00:00:01,680
Akiri Akiri

2
00:00:01,680 --> 00:00:03,520
Akiri

3
00:00:05,640 --> 00:00:08,560
This is number one The special number

4
00:00:08,560 --> 00:00:11,520
one Her shape is like a stick Say

5
00:00:11,600 --> 00:00:14,400
one everyone One, one,

6
00:00:14,400 --> 00:00:16,600
one, one There is one sound
```

## Why?

This feature is needed because subtitles improve learning outcome.

**Comment by tuancoltech** [comment]
@jo-elimu Can you modify the web app to allow uploading `SRT` for a video?
http://tgl.elimu.ai/content/multimedia/video/create

**Comment by jo-elimu** [comment]
@tuancoltech Yes. First, do you know what data format you will be needing implementing subtitles in the Android app? Just a regular `.srt` text file. Or some other format? And perhaps the default Android media player supports subtitles already?

**Comment by tuancoltech** [comment]
> [@tuancoltech](https://github.com/tuancoltech) Yes. First, do you know what data format you will be needing implementing subtitles in the Android app? Just a regular `.srt` text file, or some other format? And perhaps the default Android media player supports the display of subtitles already?

@jo-elimu We're using ExoPlayer in Android Filamu app. And it does support playing video along with subtitles.
Please refer to the supported subtitle formats here: https://developer.android.com/media/media3/exoplayer/supported-formats#standalone-subtitle

**Comment by jo-elimu** [comment]
@tuancoltech Thank you. Yes, looks like this is the exact code we need: https://developer.android.com/media/media3/exoplayer/media-items#sideloading-subtitle-tracks

So then I guess we can just refer to an `.srt` file stored somewhere on the filesystem.

Added an issue for implementing this in the backend: https://github.com/elimu-ai/webapp/issues/2061

**Comment by tuancoltech** [comment]
> [@tuancoltech](https://github.com/tuancoltech) Thank you. Yes, looks like this is the exact code we need: https://developer.android.com/media/media3/exoplayer/media-items#sideloading-subtitle-tracks
> 
> So then I guess we can just refer to an `.srt` file stored somewhere on the filesystem.
> 
> Added an issue for implementing this in the backend: [elimu-ai/webapp#2061](https://github.com/elimu-ai/webapp/issues/2061)

@jo-elimu Yes.
Further on this: When we have `.srt` files provided from backend, we can implement a separate providers for subtitles in `content-provider`, and links to it via a `subtitleId` column in `Video` table.
Then, before playing video we need to load both video data stream and subtitle stream from physical files.
